src/transport/MessageHeader.h

```diff
@@ -205,6 +216,9 @@ class MessageHeader
 
     /// Message authentication tag generated at encryption of the message.
     uint64_t mTag = 0;
+
+    // CHIP Profile ID
+    uint16_t mProfileID = 0;
```

Comment:
Should be uint32_t or we should define this as a structure with two uint16_ts, one for the vendor ID and one for the profile ID, together which form the message profile ID.